### ISSUE TITLE
Whitespace disapears after render for inline math

### ISSUE THREAD

**Issue by dinhanhthi** [goessner/markdown-it-texmath#25]
I've used the default settings. After rendering, `aaa $x$ aaa` should become `aaa x aaa`. However, it becomes `aaaxaaa`. For a temporary solution, I added a css `eq{margin: 0 5px;}` but it's not good! Any idea about this problem?

**Comment by dinhanhthi** [comment]
I think I see the problem. It comes from `<eq>` tag you use for inline math. I've changed it to `<span class="eq">` instead and everything works fine. Please consider to check again.

**Comment by grtlr** [comment]
I'm experiencing the same problem.

@dinhanhthi Where did you change this?

**Comment by dinhanhthi** [comment]
@grtlr I downloaded + modified + used it locally (i.e. inside my 11ty website). You can check the tut [here](https://dinhanhthi.com/11ty-nunjucks/#math-equations). In short, I changed `<eq>` tags to `<span>` tags and all `eq` css to `.eq` css.

**Comment by grtlr** [comment]
That’s great, thank you!

**Comment by goessner** [comment]
I am not able to reproduce with version 0.9 ...

thanks

**Comment by dinhanhthi** [comment]
@goessner I don't use 0.9, I keep using 0.8 and it works fine. What's your exact problem? And why you wanna use version 0.9?

**Comment by dinhanhthi** [comment]
@goessner I'm sorry, I don't recognize you (the author), I got you as another persone who uses this plugin. Thanks for your suggestion. If there are problems with 0.8, I will try with 0.9.

**Comment by marcinwrochna** [comment]
I had the same issue. Turns out it's not related to markdow-it-texmath, but posting here in case someone had the same. it's actually a html-minifier that removed the whitespace, probably not recognising the custom <eq> and <eqn> tags that katex produces. I had to disable `collapse_whitespace` in kangax's [html-minifier](http://perfectionkills.com/experimenting-with-html-minifier/#collapse_whitespace) that I used.
